prime-router/metadata/hl7_mapping/resources/Patient/PD1.yml

```diff
@@ -0,0 +1,43 @@
+constants:
+  hl7PD1Field: '%{hl7SegmentGroup}/PD1'
+
+elements:
+  - name: ordering-facility-name
+    #    condition: '%context.generalPractitioner.resolve().is(Organization)'
```

Comment:
Looks like we should have this check? Alternatively, this could get moved to be a `where` clause in the resource

Reply:
added it to the where clause